reference/employer/openapi.yaml

```diff
@@ -1580,10 +1580,10 @@ paths:
           `employee_deduction.amount`| `integer` | Deduction amount in cents (if `fixed`) or basis points (if `percent`)
           `company_contribution.type` | `string` | Contribution Type (`fixed` or `percent`)
           `company_contribution.amount`| `integer` | Contribution amount in cents (if `fixed`) or basis points
-          `catch_up`| `boolean` | Whether to enable catch up for this individual.
-          `annual_maximum`| `integer` (`nullable`) | The annual maximum in cents for this individual.
-          `annual_contribution_limit` | `string` | `individual` or `family`
-          `effective_date` | `string` (`nullable`) | The date which the benefit should take effect by. **This field is currently only available for assisted benefits.**
+          `catch_up`| `boolean` | Whether to enable catch up for this individual
+          `annual_maximum`| `integer` (`nullable`) | The annual maximum in cents for this individual
+          `annual_contribution_limit` | `string` (`nullable`) | Whether HSA is applied towards `individual` or `family`
```

Comment:
i think this is fine since the rest of our docs use nullable as optional, but it isn't technically correct. if a developer passes `annual_contribution_limit: null` it will fail